File: Easy/penultimate_word.py
data = ['some line with text', 'another line']

#import sys
#test_cases = open(sys.argv[1], 'r')
#data = test_cases.readlines()
for test in data:
	test = test.rstrip()
	test = test.split(" ")
	print test[len(test)-2]

#test_cases.close()

File: Easy/penultimate_word.r
test.cases <- c("some line with text",
				"another line")

#args <- commandArgs(trailingOnly=TRUE)
#test.cases <- strsplit(readLines(args[[1]], warn=FALSE), '\n')
cat(sapply(test.cases, function(test) {
	if (length(test) > 0) {
		words <- strsplit(test, " ")[[1]]
		words[length(words)-1]
	}
}), sep = "\n")